src/iceflow/itrf/converter.py

```diff
@@ -1,20 +1,22 @@
 from __future__ import annotations
 
+import calendar
 import datetime as dt
-import time
 
 import pandas as pd
 from pyproj import Transformer
 
 
 def _datetime_to_decimal_year(date):
     """Stolen from
-    https://stackoverflow.com/questions/6451655/python-how-to-convert-datetime-dates-to-decimal-years
+    https://stackoverflow.com/questions/6451655/python-how-to-convert-datetime-dates-to-decimal-years,
+    with one modification: `calendar.timegm` is used to set the epoch instead of
+    `time.mktime`, which assumes local time.
     """
 
     def sinceEpoch(date):
         # returns seconds since epoch
-        return time.mktime(date.timetuple())
+        return calendar.timegm(date.timetuple())
```

Comment:
This was a subtle bug (thanks stackoverflow!). `time.mktime` assumes the local timezone, not UTC. Since all of our timestamps are UTC, `calendar.timegm` should be used instead.